fusion-studio-extension/src/browser/widget/welcome.tsx

```diff
@@ -43,7 +43,7 @@ export class FSWelcomeWidget extends ReactWidget {
       <span className="checkbox" onClick={e => this.store(!this.checked)}>
         <span className={'checkbox-box' + (this.checked ? ' checked' : '')}>
           <span></span>
-        </span>
+        </span>ls
```

Comment:
@marmoure can you remove the `ls` from this? Give me a shout if you need help with rebase.